CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v0.0.16
```

Comment:
```suggestion
## v0.0.16 (unreleased)
```

Maybe part of release process is to remove `(unreleased)` and add the next version with it to make it more obvious when it is ok to add new features to a version without checking which version is currently released?

Reply:
Yeah, that sounds right. Also, we want the changelog file on github to be explicit about what's released and what's not. For users, I mean.